Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -1388,13 +1395,15 @@ private async Task<object> CopyObjectRequestAsync(CopyObjectRequestArgs args, Ca
     /// <exception cref="BucketNotFoundException">When bucket is not found</exception>
     /// <exception cref="ObjectNotFoundException">When object is not found</exception>
     /// <exception cref="AccessDeniedException">For encrypted copy operation, Access is denied if the key is wrong</exception>
-    private async Task CompleteMultipartUploadAsync(CompleteMultipartUploadArgs args,
+    private async Task<PutObjectResponse> CompleteMultipartUploadAsync(CompleteMultipartUploadArgs args,
         CancellationToken cancellationToken)
     {
         args.Validate();
         var requestMessageBuilder = await CreateRequest(args).ConfigureAwait(false);
         using var response = await ExecuteTaskAsync(NoErrorHandlers, requestMessageBuilder, cancellationToken)
             .ConfigureAwait(false);
+        return new PutObjectResponse(response.StatusCode, response.Content, response.Headers, "unknown",
```

Comment:
You could use `-1` for unknown instead of using a string to represent the length. Do we really need to expose the length on put object? Does the S3 put object / upload APIs return the length? To keep things simpler, would it be too much to ask the user to call a stat or something to get the length if they didn't know how much data they uploaded. Using a string like this is a code smell and is hard to remove once released.

Reply:
Done